### PR TITLE
Add feature flag to toggle Intercom

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Add new feature flag to toggle intercom per market.

Conditionally show/hide related buttons and components.

## Why?

We want to disable intercom for sweden to ease load on IEX.

**Ticket(s): []**

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1239** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1239" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1239?utm_source=stack-comment).

File: .env.example [hunk 1]
```diff
@@ -14,6 +14,7 @@ FEATURE_COMPARISON_TABLE=SE
 FEATURE_CAR_CANCELLATION=SE
 FEATURE_CAR_PRICE_MATCHING=SE
 FEATURE_AB_TEST_PURCHASE_FLOWS=SE
+FEATURE_INTERCOM=NO,DK
```
Comment (alebedev) [inline .env.example:17]: Just checking - was the idea to remove only from Sweden?

File: src/client/pages/Checkout/shared/CheckoutIntercomVariation.tsx [hunk 2]
```diff
@@ -2,26 +2,33 @@ import React from 'react'
...
-          bottom: '87px!important',
-        },
-      }}
-    />
-    <TextLinkContainer>
-      <Intercom.TextLinkVariation />
-    </TextLinkContainer>
-  </>
-)
+export const CheckoutIntercomVariation = () => {
```
Comment (alebedev) [inline src/client/pages/Checkout/shared/CheckoutIntercomVariation.tsx:14]: Can we render this conditionally instead of returning null?

**Review by alebedev — APPROVED** [review]
Looks good, some questions inside

File: .env.example (hunk 1 above)
Comment (robinandeer) [reply .env.example:17]: yes exactly

File: src/client/pages/Checkout/shared/CheckoutIntercomVariation.tsx (hunk 2 above)
Comment (robinandeer) [reply src/client/pages/Checkout/shared/CheckoutIntercomVariation.tsx:14]: hmm but then we need to add the check in a lot more places

I think this looked fine when testing it out - but it's a workaround

**Pull request merged**